### ISSUE TITLE
Test suite failure

### ISSUE THREAD

**Issue by snoyberg** [haskell/ghc-events#9]
```
$ cabal test
Package has never been configured. Configuring with default flags. If this
fails, please run configure manually.
Resolving dependencies...
Configuring ghc-events-0.4.4.0...
Preprocessing test suite 'test-versions' for ghc-events-0.4.4.0...
[1 of 4] Compiling GHC.RTS.EventTypes ( GHC/RTS/EventTypes.hs, dist/build/test-versions/test-versions-tmp/GHC/RTS/EventTypes.o )
[2 of 4] Compiling GHC.RTS.EventParserUtils ( GHC/RTS/EventParserUtils.hs, dist/build/test-versions/test-versions-tmp/GHC/RTS/EventParserUtils.o )
[3 of 4] Compiling GHC.RTS.Events   ( GHC/RTS/Events.hs, dist/build/test-versions/test-versions-tmp/GHC/RTS/Events.o )
[4 of 4] Compiling Main             ( test/TestVersions.hs, dist/build/test-versions/test-versions-tmp/Main.o )
Linking dist/build/test-versions/test-versions ...
Running 1 test suites...
Test suite test-versions: RUNNING...
test-versions: test/queens-ghc-6.12.1.eventlog: openBinaryFile: does not exist (No such file or directory)
Test suite test-versions: FAIL
Test suite logged to: dist/test/ghc-events-0.4.4.0-test-versions.log
0 of 1 test suites (0 of 1 test cases) passed.
```

**Comment by jberthold** [comment]
Which platform are you running on, @snoyberg ?
This might be because the test program hard-wires '/' as the path separator, see https://github.com/haskell/ghc-events/blob/master/test/TestVersions.hs#L21 . I cannot reproduce it on Linux.

**Comment by snoyberg** [comment]
This was on Linux. Usually these kinds of things are caused by a test file not being included in the sdist tarball.

**Comment by kvelicka** [comment]
That is indeed the case - http://hackage.haskell.org/package/ghc-events-0.4.4.0/src/

Neither of the queens event logs are present there. It doesn't look like the *.reference files are either?

**Comment by Mikolaj** [comment]
Isn't it fixed in the commit mentioned above (post-0.4.4.0)?

**Comment by kvelicka** [comment]
@Mikolaj It seems that the version uploaded to Hackage is missing some of the test files, as @snoyberg has suggested. I've just tested this - cleaned my cabal package cache and ran `cabal get ghc-events`. Here's the file list for `test/`:
```
➜  ghc-events-0.4.4.0  l test 
total 212K
drwxr-xr-x. 2 sm11kv Domain Users 4.0K Jul 24 11:20 .
drwxr-xr-x. 4 sm11kv Domain Users 4.0K Jul 24 11:20 ..
-rw-r--r--. 1 sm11kv Domain Users 6.6K Jul 24 11:20 782stop.eventlog
-rw-r--r--. 1 sm11kv Domain Users 6.7K Jul 24 11:20 783stop.eventlog
-rw-r--r--. 1 sm11kv Domain Users 102K Jul 24 11:20 mandelbrot-mmc-2011-06-14.eventlog
-rw-r--r--. 1 sm11kv Domain Users  66K Jul 24 11:20 mdlLogMPI1.eventlog
-rw-r--r--. 1 sm11kv Domain Users 6.4K Jul 24 11:20 pre77stop.eventlog
-rw-r--r--. 1 sm11kv Domain Users 1.3K Jul 24 11:20 stop.hs
-rw-r--r--. 1 sm11kv Domain Users 2.3K Jul 24 11:20 TestVersions.hs
```

Note that in addition to both queens event logs missing, none of the `*.reference` files are present either.

Could it be that the version with f7ca66a44c016d1425411488484219377a6449b1 never made it to the servers? The commit is authored on the 10th of May, but last upload to Hackage was at "Mon May 4 13:28:10 UTC 2015 by MikolajKonarski".

**Comment by Mikolaj** [comment]
@kvelicka: indeed your diagnosis is right, the commit mentioned above was never published, as it was committed after the last version (0.4.4.0) was published.

**Comment by kvelicka** [comment]
Considering that this is fixed in master, closing.

**Comment by bergmark** [comment]
The test-suite for this package has been disabled in stackage because of this issue. If a fix is published to hackage can you please ping me?

Thanks!

ping @borsboom (stackage maintainer for ghc-events)

**Comment by jberthold** [comment]
AFAICT this was fixed in [`f7ca66a4`](https://github.com/haskell/ghc-events/commit/f7ca66a44c016d1425411488484219377a6449b1#diff-363791a26da76c93622163f471748ef5) but which is not yet on hackage.

**Comment by kvelicka** [comment]
@jberthold is right. Is there a way to bump the Hackage version to that one?

**Comment by Mikolaj** [comment]
@kvelicka: AFAIK, the only way is to release a new version of the package. We are approaching that moment, aren't we?

**Comment by kvelicka** [comment]
A little more work needs to be done to make it release ready, mostly testing and cleaning everything up. That I think would suffice for a release to start 0.5.*. 

However, issues like broken log merging would have to wait until later I think. That would advance the release and I'm only aware of problems carried over from the previous versions, not introduced by new code.

**Comment by maoe** [comment]
This was fixed in v0.5.0.0 and v0.6.0 has been released on Hackage.
